extensions/panache/hibernate-orm-panache-kotlin/deployment/src/main/java/io/quarkus/hibernate/orm/panache/kotlin/deployment/KotlinPanacheEntityClassVisitor.java

```diff
@@ -52,20 +50,10 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
     }
 
     @Override
-    protected String getModelDescriptor() {
-        return "Ljava/lang/Class;";
-    }
-
-    @Override
-    protected String getPanacheOperationsBinaryName() {
+    protected String getPanacheOperationsInternalName() {
```

Comment:
I would actually like to entirely eliminate the use of "binary name" as that does not appear (that I can find) in either `asm` or `jandex` but appears to have been crafted with quarkus.  Usually the use of the different forms of these name is either as a `descriptor`  (`L/java/lang/String;`) or an internal name (`java/lang/String`).  But making that change *now* is far reaching and I didn't want to unilaterally make that decision.  I think only changed that in this case because I kept getting confused as to what that form should in this case.  But the formal/official term is "internal name" at any rate.